File: CCC/CCC 18/CCC 18 J2 - Occupy parking/Occupy_parking.py
import sys

N = int(sys.stdin.readline())
line1 = sys.stdin.readline().strip()
line2 = sys.stdin.readline().strip()

cont = 0
for i in range(N):
    if line1[i] == line2[i] == 'C':
        cont += 1

print(cont)

File: CCC/CCC 18/CCC 18 J2 - Occupy parking/Occupy_parking.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main(){
    int N, cont=0;
    char line1[101], line2[101];

    scanf("%d %s %s", &N, line1, line2);
    for (int i=0; i<N; i++){
        if ((line1[i] == 'C') && (line2[i] == 'C')){
            cont++;
        }
    }
    printf("%d", cont);
}
